app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -1,13 +1,10 @@
-/* Placeholder for backend dummy app */
 (function($) {
   SpreePayuIntegration = {
-    updateSaveAndContinueVisibility: function(e) {
+    updatePayuSelectedClass: function(e) {
```

Comment:
hold your horses! you may be right and it could work for both (TIL about the `state` parameter for `toggleClass`, will be back in few mins :) )